pyresample/_spatial_mp.py

```diff
@@ -186,19 +186,22 @@ def __init__(self, *args, **kwargs):
         pass
 
     def transform_lonlats(self, lons, lats):
-
+        """Transform longitudes and latitues to cartesian coordinates."""
+        if np.issubdtype(lons.dtype, np.integer):
+            lons = lons.astype(np.float)
         coords = np.zeros((lons.size, 3), dtype=lons.dtype)
-        deg2rad = lons.dtype.type(np.pi / 180)
         if ne:
+            deg2rad = np.pi / 180  # noqa: F841
```

Comment:
Just curious, what does flake8 have a problem with on this line?

Reply:
That we define deg2rad but then never use it .... :)

Reply:
Ah of course. thanks.